### PR TITLE
Fix/3128 (Graph for routed network DHCP)

### LINKED ISSUE #3128: pfstats DHCP stats with routed networks
Concern expressed by @satkunas in #3113:

I see a possible issue that we may need to address. We're generating DHCP leases based on `interfaces::listen_ints`. 

Is it possible to have more than one DHCP pool per `int`? If so, `pfstats`  only uses the first index from `/api/v1/dhcp/stats/[interface]` (`$[0].free`) and if this API endpoint returns more than one result for a single interface the subsequent results are ignored. Let me know if this has to be addressed and requires support for multiple DHCP servers for a single interface before merging. I'll need to add a second level of iteration within `foreach interface in listen_ints` both in the backend and in the dashboard template, or possibly switch to the `@dhcplistener_ints` - if you can confirm that this variable will work better.

### PR DESCRIPTION
**Pull request by fdurand**
# Description
DHCP Statistics for all networks.

# Impacts
Dashboard

# Issue
fixes #3128 

# Delete branch after merge
YES

# NEWS file entries
## Bug Fixes
* DHCP Statistics for routed networks missing on the dashboard (#3128)

### CODE REVIEW

File: go/database/connection.go [hunk 1]
```diff
@@ -2,14 +2,20 @@ package database
...
+	"os"
 
 	"github.com/inverse-inc/packetfence/go/pfconfigdriver"
 )
 
 func Connect(user, pass, host, port, dbname string) *sql.DB {
 	var where string
 	if host == "localhost" {
-		where = "unix(/var/lib/mysql/mysql.sock)"
+		if _, err := os.Stat("/etc/debian_version"); err == nil {
```
Comment (julsemaan) [inline go/database/connection.go]: It would be good to consolidate this function with the one in go/db/db.go to prevent duplication

**Review by julsemaan — COMMENTED** [review]
Other for my DRYing up comment, this looks good and seems to have a small impact on the code so I'd be comfortable in merging this for 8.1 if nobody objects to it

File: go/database/connection.go (hunk 1 above)
Comment (fdurand) [reply go/database/connection.go]: Do we want to do it in this PR ?

Comment (julsemaan) [reply go/database/connection.go]: Since we're already changing stuff in it, I'd say to put it in this PR

Comment (fdurand) [reply go/database/connection.go]: Ok done

**Pull request merged**